### PR TITLE
1863-community-queries

### LINKED ISSUE #1863: Add more useful queries in documentation
**Problem Report**
To understand db-sync better and get devs working faster it would be great to increase the amount of 'useful queries'. Maybe taking TapTools as a good base. It would really help to understand the relationships better. 

1. Current holdings of any address and stakekey
2. Past holdings of any address and stakekey
3. Get asset details (names, logos, etc)

Maybe also some advanced stuff. I don't know if they are possible just from db-sync?
1. Calculate prices for assets (maybe start with Minswap as an example?)
2. A query selecting only "NFTs"
3. Unique wallets trading
4. Trading volume

I'm sure there are many more useful ones.

### PR DESCRIPTION
**Pull request by Cmdv**
# Description

This fixes #1863

# Checklist

- [ ] Commit sequence broadly makes sense
- [ ] Commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] Any changes are noted in the [changelog](https://github.com/IntersectMBO/cardano-db-sync/blob/master/db-sync/CHANGELOG.md)
- [ ] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) on version 0.10.1.0 (which can be run with `scripts/fourmolize.sh`)
- [ ] Self-reviewed the diff

# Migrations

- [ ] The pr causes a [breaking change](https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/migrations.md) of type a,b or c
- [ ] If there is a breaking change, the pr includes a database migration and/or a fix process for old values, so that upgrade is possible
- [ ] Resyncing and running the migrations provided will result in the same database semantically

If there is a breaking change, especially a big one, please add a justification here. Please elaborate
more what the migration achieves, what it cannot achieve or why a migration is not possible.

### CODE REVIEW

File: doc/community-tools.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Community tools
+
+This section provides a list of community-developed tools that integrate with cardano-db-sync. We appreciate the efforts of these developers in expanding the ecosystem. If you have a tool you'd like to add to this list, please open an issue or a pull request.
+
+### Koios
+- **Description:** The Koios Artifacts repository is part of the Cardano community and acts as a collection or set of artifacts for the Koios service. Koios is an open-source, lightweight Cardano REST API that provides various endpoints to interact with the Cardano blockchain. 
+- **Repository:** [https://github.com/cardano-community/koios-artifacts](https://github.com/cardano-community/koios-artifacts)
```
Comment (rdlrt) [inline doc/community-tools.md:7]: My suggestion for an on-looker who may not be interested in entire repository would be to link directly to folders where SQLs are located (i.e. https://github.com/cardano-community/koios-artifacts/tree/main/files/grest/rpc) for Koios

File: doc/community-tools.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- **Website:** [https://koios.rest/](https://koios.rest/)
+
+### Blockfrost
+- **Description:** Blockfrost provides APIs for easy interaction with blockchain networks like Cardano. It allows developers to access blockchain data and perform operations without running their own nodes, simplifying blockchain application development.
+- **Repository:** [https://github.com/blockfrost/blockfrost-backend-ryo](https://github.com/blockfrost/blockfrost-backend-ryo)
```
Comment (rdlrt) [inline doc/community-tools.md:12]: Similar to above (location for blockfrost-ryo being https://github.com/blockfrost/blockfrost-backend-ryo/tree/master/src/sql )

Comment (rdlrt) [reply doc/community-tools.md:12]: Another one to add potentially could be Gov Tools : https://github.com/IntersectMBO/govtool/tree/develop/govtool/backend/sql

File: doc/community-tools.md (hunk 1 above)
Comment (Cmdv) [reply doc/community-tools.md:7]: I've added those direct links in `interesting-queries.md`. This file is more about the whole project. Though I could also add the SQL queries here too so it's in more than one place.

**Review by kderme — APPROVED** [review]

**Pull request merged**